### PR TITLE
[14.0] [MIG] fieldservice_location_builder

### PR DESCRIPTION
**Pull request by Murtaza-OSI**

### CODE REVIEW

File: fieldservice_location_builder/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Field Service Location Builder",
+    "summary": "Adds a tool to help build out Location Hierarchies",
+    "version": "14.0.1.0.0",
+    "category": "Field Service",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/field-service",
+    "depends": ["fieldservice_account_analytic"],
```
Comment (pedrobaeza) [inline fieldservice_location_builder/__manifest__.py]: Why this depends on `fieldservice_account_analytic`? Isn't a very hard dependency for only this tool?

File: fieldservice_location_builder/wizard/fsm_location_builder_wizard.py [hunk 2]
```diff
@@ -4,54 +4,52 @@
...
-                'fsm_parent_id': parent.id,
-                'street': location.street,
-                'street2': self.level_ids[num].name + spacer + str(lev_id),
-                'city': location.city,
-                'zip': location.zip,
-                }
+        vals = {
+            "name": self.level_ids[num].name + spacer + str(lev_id),
+            "owner_id": location.owner_id.id,
+            "customer_id": location.customer_id.id,
```
Comment (chienandalu) [inline fieldservice_location_builder/wizard/fsm_location_builder_wizard.py]: This field is provided by `fieldservice_account_analytic`. If you want a soft dependency, you can check aside if the field is available with `hassattr(location, "customer_id")` and updating the `vals` dict if so

**Comment by chienandalu** [comment]
It still has a hidden dependency. Following my tip, you can work around it without changing your code logic too much

**Review by chienandalu — APPROVED** [review]
Thanks! :slightly_smiling_face: :+1:

**Comment by Freni-OSI** [comment]
@pedrobaeza @max3903 It's ready for review

**Review by max3903 — APPROVED** [review]

**Pull request merged**